centreon/www/install/php/Update-next.php

```diff
@@ -245,6 +245,82 @@
     }
 };
 
+// -------------------------------------------- Agent Configuration -------------------------------------------- //
+/**
+ * Add prefix "/etc/pki/" and extensions (.crt, .key) to certificate and key paths in agent_configuration table.
+ *
+ * @param CentreonDB $pearDB
+ *
+ * @throws CentreonDbException
+ *
+ * @return void
+ */
+$updateAgentConfiguration = function (CentreonDB $pearDB) use (&$errorMessage): void {
+    $errorMessage = 'Unable to retrieve data from agent_configuration table';
+    $statement = $pearDB->executeQuery(
+        <<<'SQL'
+            SELECT `id`, `configuration` FROM `agent_configuration`
+        SQL
+    );
+
+    $errorMessage = 'Unable to update agent_configuration table';
+    $updates = [];
+    while ($row = $statement->fetch(\PDO::FETCH_ASSOC)) {
+        $config = json_decode($row['configuration'], true);
+        if (! is_array($config)) {
+            continue;
+        }
+
+        foreach ($config as $key => $value) {
+            if (str_ends_with($key, '_certificate') && is_string($value)) {
```

Comment:
For indempotence it would be nice to check if value is does not match /etc/pki/xxx.crt or /etc/pki/xxx.key before appling the modification